lib/rubocop/cop/rspec/expect_actual.rb

```diff
@@ -80,8 +80,6 @@ def on_send(node)
 
         private
 
-        # This is not implemented using a NodePattern because it seems
```

Comment:
It feels that this note is telling “don’t waste time to refactor”. Do you think it makes sense to remove it?